src/inst/gen_serv_nodes/consul.asc

```diff
@@ -97,3 +97,5 @@ Finally, check DNS requests on virtual zone are managed by Consul with:
 
 The output must report multiple generic service nodes static IP addresses in
 random order.
+
+To advanced custom consul services definition, please <<_custom_consul_services,go to this section...>>
```

Comment:
I really prefer unaliased internal reference (ie. `<<_custom_consul_services>>`) since we build the documents with `:xrefstyle: full` which produces links that can also be read when the document is printed.